Makefile

```diff
@@ -91,8 +86,19 @@ vet:
 # Generate code
 .PHONY: generate
 generate: $(CONTROLLER_GEN)
+	$(MAKE) generate-manifests
+	$(MAKE) generate-deepcopy
+
+# Generate deepcopy files.
+.PHONY: generate-deepcopy
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths=./api/...
 
+# Generate manifests e.g. CRD, RBAC etc.
+.PHONY: generate-manifests
+generate-manifests:
```

Comment:
This and generate-deepcopy need to depend on `$(CONTROLLER_GEN)` just like generate does. They should be able to run independently of `generate`.